src/D2Reader/GameMemoryTableFactory.cs

```diff
@@ -68,30 +68,30 @@ private GameMemoryTable CreateMemoryTableForVersion114D(IntPtr BaseAddress)
         {
             return new GameMemoryTable()
             {
-                Loading = BaseAddress + 0x30F2C0,
-                Saving = BaseAddress + 0x3792F8,
-                Saving2 = BaseAddress + 0x3786D0,
-                InGame = BaseAddress + 0x30EE8C,
-                InMenu = BaseAddress + 0x379970,
+                Loading                     = BaseAddress + 0x30F2C0,
+                Saving                      = BaseAddress + 0x3792F8,
+                Saving2                     = BaseAddress + 0x3786D0,
+                InGame                      = BaseAddress + 0x30EE8C,
+                InMenu                      = BaseAddress + 0x379970,
 
-                GlobalData = BaseAddress + 0x00344304, // game.744304
-                World = BaseAddress + 0x00483D38,
-                PlayersX = BaseAddress + 0x483D70,
-                GameId = BaseAddress + 0x00482D0C,
-                LowQualityItems = BaseAddress + 0x56CC58,
-                ItemDescriptions = BaseAddress + 0x56CA58,
-                MagicModifierTable = BaseAddress + 0x56CA7C,
-                RareModifierTable = BaseAddress + 0x56CAA0,
-                Units113 = null,
-                Units114 = BaseAddress + 0x003A5E70,
-                PlayerUnit = BaseAddress + 0x003A5E74,
-                Area = BaseAddress + 0x003A3140,
-                Pets = BaseAddress + 0x003BB5BC,
-                InventoryTab = BaseAddress + 0x3BCC4C,
-                StringIndexerTable = BaseAddress + 0x4829B4,
-                StringAddressTable = BaseAddress + 0x4829B8,
-                PatchStringIndexerTable = BaseAddress + 0x4829D0,
-                PatchStringAddressTable = BaseAddress + 0x4829BC,
+                GlobalData                  = BaseAddress + 0x344304, // game.744304
+                World                       = BaseAddress + 0x483D38,
+                PlayersX                    = BaseAddress + 0x483D70,
+                GameId                      = BaseAddress + 0x482D0C,
+                LowQualityItems             = BaseAddress + 0x56CC58,
+                ItemDescriptions            = BaseAddress + 0x56CA58,
+                MagicModifierTable          = BaseAddress + 0x56CA7C,
+                RareModifierTable           = BaseAddress + 0x56CAA0,
+                Units113                    = null,
+                Units114                    = BaseAddress + 0x3A5E70,
+                PlayerUnit                  = BaseAddress + 0x3A5E74,
+                Area                        = BaseAddress + 0x3A3140,
+                Pets                        = BaseAddress + 0x3BB5BC,
+                InventoryTab                = BaseAddress + 0x3BCC4C,
+                StringIndexerTable          = BaseAddress + 0x4829B4,
+                StringAddressTable          = BaseAddress + 0x4829B8,
+                PatchStringIndexerTable     = BaseAddress + 0x4829D0,
+                PatchStringAddressTable     = BaseAddress + 0x4829BC,
```

Comment:
there is no change here apart from the formatting
the file should not be part of the PR

maybe we can implement some kind of coding standard and then reformat all the code accordingly, but for now i'd like to keep this as it is, if there is no real code change